external/storm-kafka-monitor/src/main/java/org/apache/storm/kafka/monitor/KafkaOffsetLagUtil.java

```diff
@@ -373,16 +377,20 @@ private static Options buildOptions () {
         curatorFramework.start();
         String partitionPrefix = "partition_";
         String zkPath = oldKafkaSpoutOffsetQuery.getZkPath();
-        if (!zkPath.endsWith("/")) {
-            zkPath += "/";
+         if (zkPath.endsWith("/")) {
+            zkPath = zkPath.substring(0, zkPath.length()-1);
+        }
+        if (curatorFramework.checkExists().forPath(zkPath) == null) {
+        	System.out.printf("--zk-node '%s' is not exists.%n", zkPath);
```

Comment:
%n? You mean.. println? Also small nit: "is not exists" => "does not exist"

This is also using tabs so the indentation is off. Why not use spaces (4 spaces)?